docker/Dockerfile

```diff
@@ -36,7 +36,7 @@ RUN GOOS=linux GOARCH=$TARGETARCH make build
 RUN GOOS=linux GOARCH=$TARGETARCH make test-unit
 
 # Final image.
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.5
```

Comment:
Do we want to update UBI version now? OpenShift operator in version 1.0.0 has already been published with 8.5. With that change we will have version 1.0.0 in two variants. Maybe we can delay this update to after the release on OperatorHub?